pombola/south_africa/views.py

```diff
@@ -692,7 +701,8 @@ def store_or_get_pmg_member_id(self, scheme='za.org.pmg.api/member'):
             url_fmt = "https://api.pmg.org.za/member/?filter[pa_link]={}"
             api_search_url = url_fmt.format(pa_link)
             try:
-                search_resp = requests.get(api_search_url)
+                search_resp = requests.get(
+                    api_search_url, timeout=API_REQUESTS_TIMEOUT)
             except requests.RequestException:
```

Comment:
Should this (and other similar ones) also be requests.exceptions.RequestException?